gcc 8.1 - warning

I have tested the codebase with
* gcc 8.1.0 + `CXXFLAGS="-pedantic -Werror -fsanitize=address"` 
and I have received this interesting warning :
```log
[ 14%] Building CXX object CMakeFiles/osm2pgsql_lib.dir/node-ram-cache.cpp.o
[ 15%] Building CXX object CMakeFiles/osm2pgsql_lib.dir/options.cpp.o
[ 16%] Building CXX object CMakeFiles/osm2pgsql_lib.dir/osmdata.cpp.o
[ 17%] Building CXX object CMakeFiles/osm2pgsql_lib.dir/osmium-builder.cpp.o
/openstreetmap/osm2pgsql/osmium-builder.cpp: In member function 'geom::osmium_builder_t::wkbs_t geom::osmium_builder_t::create_polygons(const osmium::Area&)':
/openstreetmap/osm2pgsql/osmium-builder.cpp:404:22: error: catching polymorphic type 'class osmium::geometry_error' by value [-Werror=catch-value=]
     } catch (osmium::geometry_error) { /* ignored */
                      ^~~~~~~~~~~~~~
[ 18%] Building CXX object CMakeFiles/osm2pgsql_lib.dir/output-gazetteer.cpp.o
cc1plus: all warnings being treated as errors
CMakeFiles/osm2pgsql_lib.dir/build.make:302: recipe for target 'CMakeFiles/osm2pgsql_lib.dir/osmium-builder.cpp.o' failed
```

other important:
* tested with `gcc:8` official docker image ( https://hub.docker.com/r/library/gcc/tags/  )
* full travis log: https://travis-ci.org/ImreSamu/osm2pgsql/jobs/388050138
* my experimental travis code -> : https://github.com/ImreSamu/osm2pgsql/tree/travis_gcc_45678 
* job passed with : `gcc:7` , `gcc:6`, `gcc:5` , `gcc:4`   :+1:

--- a/osmium-builder.cpp
+++ b/osmium-builder.cpp
@@ -401,7 +401,7 @@ osmium_builder_t::create_polygons(osmium::Area const &area)
             ret.push_back(wkb);
         }
 
-    } catch (osmium::geometry_error) { /* ignored */
+    } catch (const osmium::geometry_error &) { /* ignored */
     }
 
     return ret;

